File: event/src/test/java/de/danoeh/antennapod/event/playback/PlaybackPositionEventTest.java
package de.danoeh.antennapod.event.playback;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class PlaybackPositionEventTest {

    @Test
    public void positionAndDurationAreStored() {
        PlaybackPositionEvent event = new PlaybackPositionEvent(30, 120);
        assertEquals(30, event.getPosition());
        assertEquals(120, event.getDuration());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/playback/PlaybackPositionEventTest.kt
package de.danoeh.antennapod.event.playback

import org.junit.Assert.assertEquals
import org.junit.Test

class PlaybackPositionEventTest {

    @Test
    fun positionAndDurationAreStored() {
        val event = PlaybackPositionEvent(30, 120)
        assertEquals(30, event.position)
        assertEquals(120, event.duration)
    }
}
